src/integration/java/io/pinecone/helpers/AssertRetry.java

```diff
@@ -10,7 +10,7 @@ public class AssertRetry {
     private static final int delay = 1500;
 
     public static void assertWithRetry(AssertionRunnable assertionRunnable) throws InterruptedException, PineconeException {
-        assertWithRetry(assertionRunnable, 2);
+        assertWithRetry(assertionRunnable, 3);
```

Comment:
Bumped the default backoff on this.

Reply:
I understand what the second parameter does here, and I also understand this increases the amount of time the asserts will wait overall. I bumped this up because we are essentially using this utility to poll for freshness in some of our data plane tests, and I was concerned it may not be waiting long enough because I was seeing occasional failures that indicated this was the case.

Reply:
Moved this back down to 2, let's keep an eye on data plane tests needing to poll for freshness and whether this works. 👍